app/src/main/java/vn/sunasterisk/music_70/SplashActivity.kt

```diff
@@ -0,0 +1,17 @@
+package vn.sunasterisk.music_70
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.os.Handler
+
+class SplashActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        Handler().postDelayed({
+            startActivity(Intent(this, MainActivity::class.java))
+            finish()
+        },2000L)
+    }
+}
```

Comment:
Tham khảo cách tạo splash screen ở đây: https://android.jlelse.eu/right-way-to-create-splash-screen-on-android-e7f1709ba154